storybooknative/src/Text/__tests__/index.test.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import * as React from "react";
+import renderer from "react-test-renderer";
+
+import Text from "../index";
+
+describe("Text", () => {
+  const text = "Children text";
+  const component = renderer
+    .create(
+      <Text type="primary" size="small">
```

Comment:
since this is a component library, it makes sense to add tests for all different combinations of props